gui/templates/rebalances.html

```diff
@@ -3,6 +3,43 @@
 {% block content %}
 {% load humanize %}
 {% if rebalances %}
+<div class="w3-container w3-padding-small">
+  <h2>Last Successful Rebalances</h2>
+  <table class="w3-table-all w3-centered w3-hoverable">
+    <tr>
+      <th>Requested</th>
+      <th>Start</th>
+      <th>Stop</th>
+      <th>Scheduled Duration</th>
+      <th>Actual Duration</th>
+      <th>Value</th>
+      <th>Fee Limit</th>
+      <th>Target PPM</th>
+      <th>Fees Paid</th>
+      <th>Last Hop Alias</th>
+      <th>Status</th>
+      <th>Hash</th>
+    </tr>
+    {% for rebalance in rebalances %}
```

Comment:
To avoid looping through all 150 twice, you could pass another list of just successful rebalances using a database query to filter the results.